spec/src/main/asciidoc/core/spi.asciidoc

```diff
@@ -401,6 +401,20 @@ public void validate(InjectionPoint injectionPoint);
 
 ==== Firing an event
 
+The method `BeanManager.getEvent()` returns an an instance of `Event<Object>` and without qualifier.
```

Comment:
Sorry, it's not exactly that: section 3.12 is applied to Event IP but  qualifiers at `Event<>` injection point are "erased" and not added if you specify new qualifiers with the 'select()' method.

So the wording:

> The method `BeanManager.getEvent()` returns an instance of `Event` with _specified type_ `java.lang.Object` and _specified qualifier_ `@Default`. 

is probably the best one.